esphome/components/remote_base/rc_switch_protocol.cpp

```diff
@@ -245,7 +248,16 @@ bool RCSwitchDumper::dump(RemoteReceiveData src) {
         buffer[j] = (out_data & (1 << (out_nbits - j - 1))) ? '1' : '0';
 
       buffer[out_nbits] = '\0';
-      ESP_LOGD(TAG, "Received RCSwitch Raw: protocol=%u data='%s'", i, buffer);
+      uint32_t m = millis();
+      uint32_t delta_time = m - LAST_MILLIS;
+      if (delta_time < 1000 && LAST_RECEIVED_DATA == out_data && LAST_RECEIVED_NBITS == out_nbits) {
+        ESP_LOGD(TAG, "Received RCSwitch Raw: protocol=%u data='%s' Repeat: %dms", i, buffer, delta_time);
```

Comment:
```suggestion
        ESP_LOGD(TAG, "Received RCSwitch Raw: protocol=%u data='%s' Repeat: %ums", i, buffer, delta_time);
```

delta_time is unsigned